src/sagemaker/amazon/factorization_machines.py

```diff
@@ -25,7 +25,10 @@
 
 
 class FactorizationMachines(AmazonAlgorithmEstimatorBase):
-    """Placeholder docstring"""
+    """A general-purpose supervised learning algorithm that can be used for both
+    classification and regression tasks. It is an extension of a linear model that
+    is designed to capture interactions between features within high dimensional
+    sparse datasets economically."""
```

Comment:
nit [pep-0257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings): one line summary, followed by blank line and then detailed content.
```suggestion
    """A supervised learning algorithm used in classification and regression.

    Factorization Machines combine the advantages of Support Vector Machines
    with factorization models. It is an extension of a linear model that is
    designed to capture interactions between features within high dimensional
    sparse datasets economically.
    """
```

Reply:
The request I made in issue #1780 was for the missing doc strings for model and predictor classes. For example, [class FactorizationMachinesModel(Model)](https://github.com/aws/sagemaker-python-sdk/blob/762c6fab77cd2628e83617ab8538cf6678a8f1d4/src/sagemaker/amazon/factorization_machines.py#L301) is having empty doc strings. Same for all other 1p algos.